pkg/version/version.go

```diff
@@ -0,0 +1,32 @@
+package version
+
+import (
+	"k8s.io/apimachinery/pkg/version"
+)
+
+var (
+	// commitFromGit is a constant representing the source version that
+	// generated this build. It should be set during build via -ldflags.
+	commitFromGit string
+	// versionFromGit is a constant representing the version tag that
+	// generated this build. It should be set during build via -ldflags.
+	versionFromGit string
+	// major version
+	majorFromGit string
+	// minor version
+	minorFromGit string
+	// build date in ISO8601 format, output of $(date -u +'%Y-%m-%dT%H:%M:%SZ')
+	buildDate string
+)
+
+// Get returns the overall codebase version. It's for detecting
+// what code a binary was built from.
+func Get() version.Info {
+	return version.Info{
+		Major:      majorFromGit,
+		Minor:      minorFromGit,
+		GitCommit:  commitFromGit,
+		GitVersion: versionFromGit,
+		BuildDate:  buildDate,
```

Comment:
Might as well fill in `GoVersion`, `Compiler`, and `Platform` as well? (copy/pasteable from here: https://github.com/operator-framework/catalogd/blob/b996f69857cc2c4ffd6d2c74a78c2974a0997a36/internal/version/version.go#L26-L28)